lib/hpke/include/hpke/certificate.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include "bytes/bytes.h"
+#include <memory>
+#include <optional>
+
+using namespace bytes_ns;
+using namespace std;
```

Comment:
Never do `using namespace std;`.  In general `using namespace` is to be avoided; we just do it for `bytes` for convenience.